apps/src/StudioApp.js

```diff
@@ -193,6 +194,18 @@ var StudioApp = function () {
    */
   this.wireframeShare = false;
 
+  /**
+   * Redux store that might be provided by the app. Initially give it an empty
+   * interface so that we can assume existence.
+   */
+  this.reduxStore_ = {
+    getState: function () {
+      return {};
+    },
+    dispatch: function () {
+    }
+  };
```

Comment:
Thoughts on providing these defaults vs. having reduxStore_ be null and making null checks everywhere?